main.py

```diff
@@ -0,0 +1,69 @@
+from typing import Optional
+
+from fastapi import Depends, Query, FastAPI, HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+from db.engine import SessionLocal
+
+app = FastAPI()
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.get("/authors/", response_model=list[schemas.Author])
```

Comment:
Ensure that the `response_model` for the `/authors/` endpoint matches the actual schema used in the application. Verify that `schemas.Author` is correctly defined and used.